src/EventSubscriber/BundleSetupSubscriber.php

```diff
@@ -37,5 +38,6 @@ public function bundleSetup(BundleSetupEvent $event): void
     {
         // add required PimcoreAdminBundle
         $event->addRequiredBundle('PimcoreAdminBundle', PimcoreAdminBundle::class, true);
+        $event->addInstallableBundle('PimcoreQuillBundle', PimcoreQuillBundle::class, true);
```

Comment:
```suggestion
        $event->addRequiredBundle('PimcoreQuillBundle', PimcoreQuillBundle::class, true);
```

Should we make it a required bundle? 
Probably every developer expects to have a WYSIWYG when installing Pimcore, right? 🤔